packages/lesswrong/components/jargon/JargonEditorRow.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { Components, getFragment, registerComponent } from '../../lib/vulcan-lib';
+import { commentBodyStyles } from '@/themes/stylePiping';
+import classNames from 'classnames';
+import { useUpdate } from '@/lib/crud/withUpdate';
+import Button from '@material-ui/core/Button';
+
+const styles = (theme: ThemeType) => ({
+  root: {
+    width: '100%',
+    display: 'flex',
+  },
+  flex: {
+    display: 'flex',
+    flexGrow: 1,
+    border: '1px solid transparent',
+    borderRadius: 4,
+    padding: 8,
+    flexDirection: 'row',
+    alignItems: 'top',
+    ...commentBodyStyles(theme),
+    marginBottom: 0,
+  },
+  toggleAndEdit: {
+    marginRight: 8,
+  },
+  isActive: {
+    border: `1px solid ${theme.palette.greyAlpha(0.5)}`,
+  },
+  input: {
+    flexGrow: 1,
+    marginRight: 8,
+  },
+  toggleSwitch: {
+  },
+  editButton: {
+    textWrap: 'nowrap',
+  },
+  deleteButton: {
+    opacity: 0,
+    '$root:hover &': {
+      opacity: 1
+    },
+    minHeight: "auto !important",
+    minWidth: "auto !important",
+    height: 24,
+    marginTop: 8,
+    width: 16,
+    marginRight: -8,
+    marginLeft: 4,
+  },
+});
+
+export const JargonEditorRow = ({classes, jargonTerm}: {
+  classes: ClassesType<typeof styles>,
+  jargonTerm: JargonTermsFragment,
+}) => {
+  const { ToggleSwitch, WrappedSmartForm, ContentItemBody } = Components;
+
+  const [isActive, setIsActive] = React.useState(!jargonTerm.rejected);
+  const [edit, setEdit] = React.useState(false);
+  const [hidden, setHidden] = React.useState(false);
+
+  const {mutate: updateJargonTerm} = useUpdate({
+    collectionName: "JargonTerms",
+    fragmentName: 'JargonTermsFragment',
+  });
+
+  const handleActiveChange = (value: boolean) => {
+    setIsActive(value);
+    void updateJargonTerm({
+      selector: { _id: jargonTerm._id },
+      data: {
+        rejected: !value
+      },
+    })
+  }
+
+  const handleDelete = () => {
+    setHidden(true);
+    void updateJargonTerm({
+      selector: { _id: jargonTerm._id },
+      data: {
+        deleted: true
+      },
+    })
+  }
+
+  if (hidden) return null;
+
+  return <div className={classes.root}>
+    <div className={classNames(classes.flex, isActive && classes.isActive)}>
+      <div className={classes.toggleAndEdit}>
+        <ToggleSwitch value={isActive} className={classes.toggleSwitch} setValue={handleActiveChange}/>
+        {isActive && <a className={classes.editButton} onClick={() => setEdit(!edit)}>Edit</a>}
+      </div>
+      {!isActive && <div dangerouslySetInnerHTML={{__html: jargonTerm.term}} />}
```

Comment:
Is this meant to be a div (vs. the isActive but not edit case, where it's `ContentItemBody`)?

Reply:
(maybe so, if the reason this is here is for future LaTeX cases)